templates/gov/index.html

```diff
@@ -0,0 +1,484 @@
+          </div>
+          <div class="icon-container__text">
+            FIPS 140-2 Level 1 certification for cryptographic modules on Ubuntu 16.04 LTS and Ubuntu 18.04 LTS
+          </div>
+        </td>
+      </tr>
+      <tr>
+        <td><strong>DISA STIG</strong></td>
+        <td>
+          <div class="icon-container__icon">
+            <i class="p-icon--dark-tick"></i>
+          </div>
+          <div class="icon-container__text">
+            Defense Information System Agency (DISA) Security Technical Implementation Guide (STIG) for Ubuntu 16.04 LTS and Ubuntu 18.04 LTS.
+          </div>
+        </td>
+      </tr>
+      <tr>
+        <td><strong>Common Criteria EAL2 certified</strong></td>
+        <td>
+          <div class="icon-container__icon">
+            <i class="p-icon--placeholder"></i>
+          </div>
+          <div class="icon-container__text">
+            Common Criteria (CC) EAL2 certification for Ubuntu 16.04 Final certification from CSEC is pending for Ubuntu 18.04.
+          </div>
+        </td>
+      </tr>
+      <tr>
+        <td><strong>CIS Benchmarks</strong></td>
+        <td>
+          <div class="icon-container__icon">
+            <i class="p-icon--placeholder"></i>
+          </div>
+          <div class="icon-container__text">
+            CIS has published benchmarks for Ubuntu 12.04 LTS and Ubuntu 14.04 LTS, with Ubuntu 16.04 LTS and Ubuntu 18.04 LTS benchmarks drafted.
+          </div>
+        </td>
+      </tr>
+      <tr>
+        <td><strong>ISO 27001, ISO 27002 certified</strong></td>
+        <td>
+          <div class="icon-container__icon">
+            <i class="p-icon--dark-tick"></i>
+          </div>
+        </td>
+      </tr>
+      <tr>
+        <td><strong>SOC2 Type 2 certified</strong></td>
+        <td>
+          <div class="icon-container__icon">
+            <i class="p-icon--dark-tick"></i>
+          </div>
+        </td>
+      </tr>
+      <tr>
+        <td><strong>FedRAMP</strong></td>
+        <td>
+          <div class="icon-container__icon">
+            <i class="p-icon--dark-tick"></i>
+          </div>
+          <div class="icon-container__text">
+            (Federal Risk and Authorization Management Program)
+          </div>
+        </td>
+      </tr>
+      <tr>
+        <td><strong>HITECH</strong></td>
+        <td>
+          <div class="icon-container__icon">
+            <i class="p-icon--dark-tick"></i>
+          </div>
+          <div class="icon-container__text">
+            (Health Information Technology for Economic and Clinical Health Act)
+          </div>
+        </td>
+      </tr>
+      <tr>
+        <td><strong>GDPR Compliance tested</strong></td>
+        <td>
+          <div class="icon-container__icon">
+            <i class="p-icon--dark-tick"></i>
+          </div>
+          <div class="icon-container__text">
+            <a href="/legal/data-privacy">Read our data privacy policy&nbsp;&rsaquo;</a>.
```

Comment:
IMO it looks weird to have this link followed by a period, given that there is no non-link copy, and the link is also followed by a right angle quote/arrow symbol